gettingstarted/InstallingEquella.md

```diff
@@ -44,24 +51,26 @@ The primary log to watch is located under ```{install_location}/logs/resource-ce
 and/or
 ~$ ./manager start
 ```
-To stop openEQUELLA or the manager, run the same command to start the application, but pass in 'stop' instead of 'start'.
+To stop openEQUELLA, run the same command to start the application, but pass in 'stop' instead of 'start'.
 
-### Windows
-Assuming openEQUELLA and the openEQUELLA Manager have been installed as services, navigate to your services, and select openEQUELLA to start.
+**Note:** The primary log to watch is located under: 
+
+{install_location}/logs/resource-centre/{current_date}/application.{X.}html
 
-## Accessing openEQUELLA / Finish Setup
-Use your browser to login to the openEQUELLA manager (default is <http://localhost:3000>)
-From here you can click the "start" button to start the openEQUELLA app server if not already done from a Windows service or \*nix start script.
 
-Once openEQUELLA has started you can log in to the server admin pages using the hostname and port you configured in the installer.
 
-The installation process will need to be completed in the server admin pages.  Please access the server admin pages, configure the fields presented, and once presented with the database screen, initialize the database.
+## Accessing openEQUELLA / Finish Setup
+Once openEQUELLA has started you can log into the server admin pages using the hostname and port you configured in the installer.
+
```

Comment:
Should we perhaps give an example with the defaults. e.g:

If you used the default of host `localhost` and port `3000`, then you'd go to <http://locahost:3000/> in your browser.

Reply:
The instruction for this now refer to the server admin page, not the manager page. I thought it was better for that to be the user's first interaction with the oEQ application, rather than the manager.
For the server admin page we don't provide any defaults